websites/S/Sonic Retro/presence.ts

```diff
@@ -0,0 +1,279 @@
+const presence = new Presence({
+		clientId: "970743721404530798"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search, hostname } = window.location;
+	switch (hostname) {
+		case "sonicretro.org": {
+			presenceData.smallImageKey = "home_512";
+			presenceData.smallImageText = "sonicretro.org";
+			if (search) {
+				presenceData.largeImageKey = "search";
+				if (search === "?s=")
+					presenceData.details = "Searching for something...";
+				else {
+					presenceData.details = `Searching for "${
+						document.title
+							.split("You searched for ")[1]
+							.split(" - Sonic Retro")[0]
+					}"`;
+				}
+			} else if (
+				(pathname === ("/" || "/#") && !search) ||
```

Comment:
this is not valid JS

Reply:
> this is not valid JS

Well then how do I go through with this line without making another `else if` statement that does the same thing; how would I rewrite this?

Reply:
you should use a logical or properly